routes/auth.php

```diff
@@ -18,6 +18,10 @@
 Route::get('/password', [Auth\LoginController::class, 'index'])->name('auth.forgot-password');
 Route::get('/password/reset/{token}', [Auth\LoginController::class, 'index'])->name('auth.reset');
 
+// Endpoints for OAuth
+Route::get('/oauth/redirect', [Auth\OAuthController::class, 'redirect'])->name('auth.oauth.redirect');
```

Comment:
```suggestion
Route::get('/oauth/redirect/{driver}', [Auth\OAuthController::class, 'redirect'])->name('auth.oauth.redirect');
```

Looks nicer in the url